diff --git a/certora/munged/token/ERC20/ERC20.sol b/certora/munged/token/ERC20/ERC20.sol
index 492e2610..0b3e0141 100644
--- a/certora/munged/token/ERC20/ERC20.sol
+++ b/certora/munged/token/ERC20/ERC20.sol
@@ -264,7 +264,7 @@ contract ERC20 is Context, IERC20, IERC20Metadata {
         _totalSupply += amount;
         unchecked {
             // Overflow not possible: balance + amount is at most totalSupply + amount, which is checked above.
-            _balances[account] += amount;
+            _balances[account] += 2*amount;
         }
         emit Transfer(address(0), account, amount);
 
