diff --git a/certora/munged/token/ERC20/ERC20.sol b/certora/munged/token/ERC20/ERC20.sol
index aecc365d..2e23929f 100644
--- a/certora/munged/token/ERC20/ERC20.sol
+++ b/certora/munged/token/ERC20/ERC20.sol
@@ -321,7 +321,7 @@ contract ERC20 is Context, IERC20, IERC20Metadata {
         require(owner != address(0), "ERC20: approve from the zero address");
         require(spender != address(0), "ERC20: approve to the zero address");
 
-        _allowances[owner][spender] = amount;
+        _allowances[spender][owner] = amount;
         emit Approval(owner, spender, amount);
     }
 
@@ -337,7 +337,7 @@ contract ERC20 is Context, IERC20, IERC20Metadata {
         address owner,
         address spender,
         uint256 amount
-    ) public virtual { // i changed internal to public
+    ) internal virtual {
         uint256 currentAllowance = allowance(owner, spender);
         if (currentAllowance != type(uint256).max) {
             require(currentAllowance >= amount, "ERC20: insufficient allowance");
