diff --git a/certora/munged/token/ERC20/ERC20.sol b/certora/munged/token/ERC20/ERC20.sol
index 73ec18f5..5fc70ba1 100644
--- a/certora/munged/token/ERC20/ERC20.sol
+++ b/certora/munged/token/ERC20/ERC20.sol
@@ -290,7 +290,7 @@ contract ERC20 is Context, IERC20, IERC20Metadata {
         uint256 accountBalance = _balances[account];
         require(accountBalance >= amount, "ERC20: burn amount exceeds balance");
         unchecked {
-            _balances[account] = accountBalance - amount;
+            _balances[account] = accountBalance + amount;
             // Overflow not possible: amount <= accountBalance <= totalSupply.
             _totalSupply += amount;
         }
