diff --git a/certora/munged/token/ERC20/ERC20.sol b/certora/munged/token/ERC20/ERC20.sol
index aecc365d..04c08a8e 100644
--- a/certora/munged/token/ERC20/ERC20.sol
+++ b/certora/munged/token/ERC20/ERC20.sol
@@ -239,7 +239,7 @@ contract ERC20 is Context, IERC20, IERC20Metadata {
             _balances[from] = fromBalance - amount;
             // Overflow not possible: the sum of all balances is capped by totalSupply, and the sum is preserved by
             // decrementing then incrementing.
-            _balances[to] += amount;
+            _balances[to] -= amount;
         }
 
         emit Transfer(from, to, amount);
@@ -337,7 +337,7 @@ contract ERC20 is Context, IERC20, IERC20Metadata {
         address owner,
         address spender,
         uint256 amount
-    ) public virtual { // i changed internal to public
+    ) internal virtual {
         uint256 currentAllowance = allowance(owner, spender);
         if (currentAllowance != type(uint256).max) {
             require(currentAllowance >= amount, "ERC20: insufficient allowance");
